macrosynergy/pnl/transaction_costs.py

```diff
@@ -0,0 +1,255 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from typing import List, Union, Tuple, Optional, Dict, Callable
+from numbers import Number
+import functools
+from macrosynergy.management.simulate import make_test_df
+from macrosynergy.download.transaction_costs import download_transaction_costs, get_fids
+from macrosynergy.management.utils import (
+    reduce_df,
+    get_cid,
+    get_xcat,
+    ticker_df_to_qdf,
+    qdf_to_ticker_df,
+)
+from macrosynergy.management.types import QuantamentalDataFrame
+
+
+def check_df_for_txn_stats(
+    df: QuantamentalDataFrame,
+    fids: List[str],
+    tcost_n: str,
+    rcost_n: str,
+    size_n: str,
+    tcost_l: str,
+    rcost_l: str,
+    size_l: str,
+) -> None:
+    expected_tickers = [
+        f"{_fid}{txn_ticker}"
+        for _fid in fids
+        for txn_ticker in [tcost_n, rcost_n, size_n, tcost_l, rcost_l, size_l]
+    ]
+    found_tickers = list(set(df["cid"] + "_" + df["xcat"]))
+    if not set(expected_tickers).issubset(set(found_tickers)):
+        raise ValueError(
+            "The dataframe is missing the following tickers: "
+            + ", ".join(set(expected_tickers) - set(found_tickers))
+        )
+
+    return None
```

Comment:
Isn't the `return None` statement redundant? Given it is the default behaviour if no return is specified.